src/containers/Header/style.scss

```diff
@@ -28,6 +28,16 @@ header {
           color: black;
           text-decoration: none;
         }
+
+        &__right {
+          margin-left: auto;
+        }
```

Comment:
no need to have BOTH, have one as the default one and then the other so we don't just add one more classname, it's a modifier of BEM --right